### ISSUE TITLE
Tweak brand config

### ISSUE THREAD

**Issue by AoElite** [GrimAnticheat/Grim#832]
Change `brand.ignored-clients` to `brand.ignored` & reword comment in configuration to make it more clear what this setting does.

*This issue is just a reminder, hence the Todo tag*

**Comment by MWHunter** [comment]
Yeah, some people think we exempt players with these brands so it should likely be changed
